src/main/java/net/earthcomputer/clientcommands/command/FramerateCommand.java

```diff
@@ -0,0 +1,45 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.Minecraft;
+import net.minecraft.network.chat.Component;
+
+import java.util.function.IntSupplier;
+
+import static com.mojang.brigadier.arguments.IntegerArgumentType.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class FramerateCommand {
+
+    public static final IntSupplier MAX_REFRESH_RATE = () -> Minecraft.getInstance().virtualScreen.screenManager.monitors.values().stream()
```

Comment:
Why not make this a method?

Reply:
I intended to make it come across as a constant that only rarely changes.

Reply:
That's not a good reason to not use a method

Reply:
What's wrong with a supplier though?